### PR TITLE
Change font

### PR DESCRIPTION
**Pull request by rivercory**
Changed font and replaced it with a modern design.

### CODE REVIEW

**Comment by rivercory** [comment]
@sledgehammer999 Please review my PR 😢

File: _site/css/style.css [hunk 1]
```diff
@@ -3,10 +3,12 @@
  * Copyright (c) 2006
  */
 
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
```
Comment (Chocobo1) [inline _site/css/style.css:6]: We intentionally prohibit linking to 3rd party resources, to avoid user tracking. This cannot be done like this.

**Review by Chocobo1 — CHANGES_REQUESTED** [review]

File: _site/css/style.css (hunk 1 above)
Comment (rivercory) [reply _site/css/style.css:6]: If so, should I change that to a local file?

Comment (Chocobo1) [reply _site/css/style.css:6]: I don't mind. But is it really worth it? Is it much better than the universal fonts?

File: _site/css/TLbanner.css [hunk 2]
```diff
@@ -61,3 +61,9 @@
     width: 200px;
   }
 }
+
+@media screen and (max-width: 700px) {
+  #forkongithub {
+    display: none;
+  }
+}
```
Comment (Chocobo1) [inline _site/css/TLbanner.css:69]: Doesn't do what the commit is describing. Move it to its own commit.

File: _site/css/style.css (hunk 1 above)
Comment (rivercory) [reply _site/css/style.css:6]: Sure! When I tested it, the readability was greatly improved.

File: _site/css/TLbanner.css (hunk 2 above)
Comment (rivercory) [reply _site/css/TLbanner.css:69]: Okay

File: _site/css/style.css (hunk 1 above)
Comment (Chocobo1) [reply _site/css/style.css:6]: If possible, could you provide a comparison screenshot?

Comment (rivercory) [reply _site/css/style.css:6]: - Before
2024-07-03 23 33 36 www qbittorrent org 297a425cfbd3
- After
2024-07-03 23 33 06 localhost fef85aa558d9

Comment (rivercory) [reply _site/css/style.css:6]: @Chocobo1 What do you think?

Comment (Chocobo1) [reply _site/css/style.css:6]: It's different... but I don't think it is better.

Comment (rivercory) [reply _site/css/style.css:6]: Why?

Comment (Chocobo1) [reply _site/css/style.css:6]: I don't like the new style.

Comment (rivercory) [reply _site/css/style.css:6]: Please explain in detail why you don't like it. Don't you like the readability? Or doesn't match the style of the project?

Comment (Chocobo1) [reply _site/css/style.css:6]: The font for TLbanner.css could be changed with another but I'll prefer using builtin fonts. If all of the builtin fonts are not good then I won't bother with a 3rd party font. It doesn't seems significant enough to add another resource just for it.

Comment (Chocobo1) [reply _site/css/style.css:6]: Another thing. I noticed the characters have larger vertical spacing and I think it is nice.
Everything else doesn't look good.

**Pull request closed without merging**